### PR TITLE
Feature/add position parameters

### PR DESCRIPTION
**Pull request by andrewstuart**

### CODE REVIEW

File: positions.go [hunk 1]
```diff
@@ -14,7 +16,35 @@ type Position struct {
...
+}
+
+// PositionParams encapsulates parameters known to the RobinHood positions API
+// endpoint.
+type PositionParams struct {
+	NonZero bool
+}
+
+// Encode returns the query string associated with the requested parameters
+func (p PositionParams) Encode() string {
```
Comment (damag) [inline positions.go]: I don't think this needs to be an exported method?
```suggestion
func (p PositionParams) encode() string {
```

File: positions.go [hunk 2]
```diff
@@ -14,7 +16,35 @@ type Position struct {
...
+
+// GetPositionsParams returns all the positions associated with a count, but
+// passes the encoded PositionsParams object along to the RobinHood API as part
+// of the query string.
+func (c Client) GetPositionsParams(a Account, p PositionParams) ([]Position, error) {
+	u, err := url.Parse(a.Positions)
+	if err != nil {
+		return nil, err
+	}
+	u.RawQuery = p.Encode()
```
Comment (damag) [inline positions.go]:
```suggestion
	u.RawQuery = p.encode()
```

File: positions.go [hunk 3]
```diff
@@ -14,7 +16,35 @@ type Position struct {
...
+
 	var r struct{ Results []Position }
-	err := c.GetAndDecode(a.Positions, &r)
+	err := c.GetAndDecode(u.String(), &r)
```
Comment (damag) [inline positions.go]: err is already declared above.
```suggestion
	err = c.GetAndDecode(u.String(), &r)
```

**Review by damag — APPROVED** [review]
This looks good to me, just have a few suggestions below.

**Comment by andrewstuart** [comment]
Tagged as 1.4.0 😄

**Comment by damag** [comment]
Awesome, thank you.

**Pull request merged**